serving/docker/scripts/create_venv.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -ex
+VENV_DIR=$1
+VENV_NAME=$2
+
+python3 -m venv ${VENV_DIR}/${VENV_NAME} --system-site-packages
+if [[ ${VENV_NAME} == "tgi" ]]; then
+    git clone https://github.com/OlivierDehaene/transformers.git
```

Comment:
This is no longer required https://github.com/huggingface/text-generation-inference/blob/abd58ff82c37d5e4f131abdac3d298927a815604/server/pyproject.toml#L29
Instead we should install [flash-attention](https://github.com/huggingface/text-generation-inference/blob/main/server/Makefile-flash-att)